scrapers/StateOfTexas.mjs

```diff
@@ -1,32 +1,53 @@
 import fetch from 'node-fetch';
 import moment from 'moment';
 
+const baseUrl = 'https://api.tnris.org/api/v1';
+let collectionIds = null;
+
 export default ({ database, DataScraper }) => {
-	return DataScraper(database, 'StateOfTexas', async () => {
-		const data = [];
-		const baseUrl = 'https://api.tnris.org/api/v1';
-		const id = '2679b514-bb7b-409f-97f3-ee3879f34448';
-		const collections = `collections/${id}`;
-		const resources = `resources?collection_id=${id}`;
-		const collectionsRequest = await fetch(`${baseUrl}/${collections}`);
-		const collection = await collectionsRequest.json();
-		const resourcesRequest = await fetch(`${baseUrl}/${resources}`);
-		const resource = await resourcesRequest.json();
+  return DataScraper(database, 'StateOfTexas', async () => {
+   
+    // Initializing idArray if it has not been initialized yet
+    if (!collectionIds) {
+      
+      // Fetching collection_id data from the API and mapping the results to an array
+      const idData = await (await fetch('https://api.tnris.org/api/v1/collections_catalog/')).json();
+      collectionIds = idData.results.map(function(item) { return item["collection_id"]; });
+    }
+
+    const data=[];
```

Comment:
Please use proper whitespace around operators to be more consistent with the file, and the rest of the code base.